components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/listener/ClaimMetadataManagementAuditLogger.java

```diff
@@ -211,4 +205,27 @@ private String buildLocalClaimData(Event event) {
         }
         return data.toString();
     }
+
+    /**
+     * Get the initiator for audit logs.
+     *
+     * @param tenantDomain Tenant domain of the initiator.
+     * @return initiator.
+     */
+    private String getInitiator(String tenantDomain) {
+
+        String initiator = null;
```

Comment:
we shouldn't initiate null here.  Instead we should make the method return Optional

Reply:
This method will not return null in any scenario. Refactored the method.